src/lazy/encoder/binary/v1_0/container_writers.rs

```diff
@@ -22,43 +20,31 @@ pub struct BinaryContainerWriter_1_0<'value, 'top> {
     // The buffer containing the parent's encoded body. When this list writer is finished encoding
     // its own data, a header will be written to the parent and then the list body will be copied
     // over.
-    parent_buffer: &'value mut BumpVec<'top, u8>,
+    buffer: &'value mut BumpVec<'top, u8>,
```

Comment:
🗺️ Removal of the `[List/SExp]ValuesWriter*` and `StructFieldsWriter` family of types meant that this variable name was no longer ambiguous.